build/yaml/testScenarios/runFunctionalTests.yml

```diff
@@ -0,0 +1,27 @@
+parameters:
+  resourceGroup: ''
+
+steps:
+- task: AzureCLI@1
+  displayName: 'Create DirectLine Channel'
```

Comment:
Shouldn't the creation of the directline client be part of the deploy portion that creates the bot registrations?

Reply:
We could create the connection with DirectLine in the deploy stage assuming the bot will always execute DirectLine tests.

Reply:
OK, created draft issue #214 so we do this later.